tilequeue/command.py

```diff
@@ -1075,6 +1072,26 @@ def delete_tile_of_interest(s3_parts, coord_ints):
     logger.info('Removing %s tiles from TOI and S3 ... done',
                 len(toi_to_remove))
 
+    logger.info('Computing tiles to add ...')
+    toi_to_add = new_toi - tiles_of_interest
+    logger.info('Computing tiles to add ... done. %s found',
+                len(toi_to_add))
+
+    logger.info('Adding %s tiles to TOI and SQS ...',
+                len(toi_to_add))
+
+    def add_tile_of_interest(coord_int):
+        # Add to Redis
+
+        # Enqueue a render request for the tile
+        pass
+
+    for coord_int in toi_to_add:
+        add_tile_of_interest(coord_int)
```

Comment:
Flow looks good to me. Are we going to be enqueueing the coordinates with custom priorities? Or like normal based on zoom? If normal, there are some existing utilities that should make it easier.